app/models/user.rb

```diff
@@ -9,6 +9,8 @@ class User < ApplicationRecord
   belongs_to :organisation, optional: true
   has_many :owned_lettings_logs, through: :organisation, dependent: :delete_all
   has_many :managed_lettings_logs, through: :organisation
+  has_many :owned_sales_logs, through: :organisation, dependent: :delete_all
```

Comment:
Does this imply that if we delete the user we delete their logs?

Reply:
I can't delete a user or an organisation in order to tests this - it's throwing an error when I attempt to do so through the database directly. Do I need to do it through the rails console?

Reply:
What error do you get? (But yes doing so through the rails console is also a valid test)

Reply:
Could also write a spec for it